packages/fiori-elements-writer/test/lrop.test.ts

```diff
@@ -135,6 +135,26 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
                 service: v2Service
             } as FioriElementsApp<LROPSettings>
         }
+        ,
+        {
+            name: 'lropV2_ts',
+            config: {
+                ...Object.assign(feBaseConfig('felrop4'), {
```

Comment:
Can you change this one to 'lropV2_ts' as well. (not critical but confused me when looking at the generated project).

Reply:
Done